### DISCUSSION TITLE
Welcome to sweep Discussions!

### DISCUSSION THREAD

**Discussion by wwzeng1** [sweepai/sweep#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AxP1999** [comment]
My name is Ayush and I want to see new ideas and questions being shared by members.

**Reply by wwzeng1** [reply]
Welcome Ayush!

**Reply by Leojack11** [reply]
Yeah

**Reply by Leojack11** [reply]
I am leojack, from Africa 🌍 looking forward learn from you guys because I am new here and I find it difficult to coding

**Reply by 996cheng** [reply]
有无聊天交流小程序开源代码啊

**Comment by progremir** [comment]
Hey! Is it allowed to self-host sweep on our own servers?
The issue here being that I cannot allow our codebase to be stored somewhere else.

 I don't quite understand the license limitations here.

**Reply by Kamamatoshy** [reply]
So I am really struggling with coding abd would like to know all the ins snd ours in case I switch careers and can be in it or something also my husband's bee coding an old phone number he said he got rid as well as my phone I feel I'd like to learn for my kiddos amd a Carre path

**Reply by wwzeng1** [reply]
Unfortunately it's not possible at the moment. We're focused on our hosted product right now.

**Comment by IgorFreik** [comment]
Hello, everyone! Firstly, great work, I love how documented everything is and I learned so much from your blog posts! 
Also, I have a question: didn't find it anywhere, how exactly is the codebase structure used for code searching? Is there a code file I missed maybe?
